packages/babel-preset-gatsby/src/index.js

```diff
@@ -62,19 +64,44 @@ module.exports = function preset(_, options = {}) {
       ],
     ],
     plugins: [
+      [
+        resolve(`@babel/plugin-transform-destructuring`),
+        {
+          loose: false,
+          selectiveLoose: [
+            `useState`,
+            `useEffect`,
+            `useContext`,
+            `useReducer`,
+            `useCallback`,
+            `useMemo`,
+            `useRef`,
+            `useImperativeHandle`,
+            `useLayoutEffect`,
+            `useDebugValue`,
+          ],
+        },
+      ],
       [
         resolve(`@babel/plugin-proposal-class-properties`),
         {
           loose: true,
         },
       ],
+      [
+        resolve(`@babel/plugin-proposal-object-rest-spread`),
```

Comment:
Is this needed? AFAIK @babel/preset-env already includes this.

Reply:
Yeah it's indeed included but AFAIK you need to explicitly include it to set the`useBuiltIns` option. But might not be needed since we have set `loose: true` in the `preset-env`, couldn't find anything about it in the docs though 🤔